### ISSUE TITLE
Two rules are reporting on Run Keyword Unless

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#945]
We should consider limiting it to just one rule:

```
keywords.robot:71:5 [W] 0319 'Run Keyword Unless' is deprecated since Robot Framework version 5.*, use 'IF' instead (deprecated-statement)
keywords.robot:71:5 [I] 0908 'Run Keyword Unless' can be replaced with IF block since Robot Framework 4.0 (if-can-be-used)
```

Not sure currently if it's bug.

**Comment by bhirsz** [comment]
The main difference is that we report deprecated-statement since Robotidy 2.0 but we filter out messages depending on configuration:

```
   deprecated_keywords = {
        "runkeywordunless": (5, "IF"),
        "runkeywordif": (5, "IF"),
        (...)
```

``if-can-be-used`` reports since 4.0 and ``deprecated-statement`` since 5.0. I guess the main difference is that Run Keyword If etc were not deprecated before, it was just suggestion to replace the syntax. So we can:

 - between 4 and 5 RF version report only ``if-can-be-used`` (Info)
- starting from 5 RF version report only ``deprecated-statement`` (Warning)

**Comment by mnojek** [comment]
> So we can:
> 
> * between 4 and 5 RF version report only `if-can-be-used` (Info)
> * starting from 5 RF version report only `deprecated-statement` (Warning)

Agreed, seems like a good approach.
